README.md

```diff
@@ -1,6 +1,6 @@
 # Builder Docker Image
 
-This project contains the docker image used to build [smartcontract/chainlink](https://github.com/smartcontractkit/chainlink).
+This project contains the docker image used to build [smartcontract/chainlink](https://hub.docker.com/r/smartcontract/builder).
```

Comment:
Hmm, ambiguous, this sentence is saying that this builder image is used for building chainlink. But yes this repo does produce the builder image on docker hub.

Reply:
How about ```
This project contains the docker image used to build [smartcontract/chainlink](https://github.com/smartcontractkit/chainlink). On merges to master it publishes a [docker image](https://hub.docker.com/r/smartcontract/builder).

Reply:
The diffeference between smartcontractkit on github and smartcontract on docker caught us here, so I used your wording, but tried to avoid the ambiguity and just link to what we were talking about.